src/LaunchDarkly.OpenFeature.ServerProvider/Provider.cs

```diff
@@ -39,7 +40,7 @@ public Provider(ILdClient client, ProviderConfiguration config = null)
         {
             _client = client;
             var logConfig = (config?.LoggingConfigurationFactory ?? Components.Logging())
-                .CreateLoggingConfiguration();
+                .Build(null);

```

Comment:
This is extremely questionable. I've not determined if there is anything else I can really do about it.